tests/conftest.py

```diff
@@ -59,17 +63,43 @@ def testdata_cache_path() -> str:
     return TESTDATA_CACHE_PATH
 
 
+@pytest.fixture(
+    params=[
+        PYTHON_VERSION,
+        pytest.param(
+            MIN_PYTHON_VERSION,
+            marks=pytest.mark.skipif(
+                MIN_PYTHON_VERSION >= PYTHON_VERSION,
+                reason="requires lower Python version",
+            ),
+        ),
+        pytest.param(
+            MAX_PYTHON_VERSION,
+            marks=pytest.mark.skipif(
+                MAX_PYTHON_VERSION <= PYTHON_VERSION,
+                reason="requires higher Python version",
```

Comment:
this will skip testing mlserver main process on py 3.9 and worker on py 3.10. this is probably safer, just curious if you spotted any issues though with this combination?